gradle.properties

```diff
@@ -24,6 +24,7 @@ namedVolumeNameTest=valtimo-test-fixtures
 org.gradle.daemon=true
 org.gradle.jvmargs=-Xmx6g -XX:MaxPermSize=2048m -XX:+HeapDumpOnOutOfMemoryError
 org.gradle.parallel=true
+org.gradle.workers.max=3
```

Comment:
Please note that the default is 'the number of CPU processors/cores'. This setting could become a problem on smaller machines. Doubt that any developer machine has less than 4 cores, but an EC2 machine might.

Reply:
Thanks for the note, indeed not fixing as we configure the CI/CD instances ourselves.